packages/babel-preset-default/index.js

```diff
@@ -8,11 +8,8 @@ module.exports = function( api ) {
 				targets: {
 					browsers: [ 'extends @wordpress/browserslist-config' ],
 				},
-				useBuiltIns: 'usage',
-			} ],
-			isTestEnv && [ '@babel/preset-env', {
-				useBuiltIns: 'usage',
 			} ],
+			isTestEnv && [ '@babel/preset-env' ],
```

Comment:
This is a breaking change for this package, should we update the changelog accordingly?

Reply:
Yes.

Reply:
We can do some cleanup to the `get-babel-config.js` file and remove the override of `useBuiltIns: false,`